lib/Test/Nginx/Util.pm

```diff
@@ -1086,6 +1086,22 @@ _EOC_
         $listen_opts .= " reuseport";
     }
 
+    my $keepalive_timeout=68000;
+    if (use_http3($block)) {
+        $keepalive_timeout = ${QuicIdleTimeout};
+        if ($block->quic_max_idle_timeout) {
+            $keepalive_timeout = $block->quic_max_idle_timeout;
+        }
+ 
+        if ($UseValgrind) {
+            $keepalive_timeout += $ValgrindExtraTimeout;
```

Comment:
the unit of `$ValgrindExtraTimeout` is 'second' here

Reply:
that's OK. we convert to ms in the next line.